cluster-logging/clo-sub.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging
+spec:
+  channel: "4.2"
+  installPlanApproval: Automatic
+  name: cluster-logging
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: clusterlogging.4.2.13-201912230557
```

Comment:
```suggestion
```
I see that there is already a version `4.2.14-202001061701` available - it will be better to not specify the `startingCSV` so we will avoid upgrades done by OLM immediately after the installation